unicon_runner/executor/sandbox.py

```diff
@@ -23,47 +16,39 @@ class SandboxExecutor(UnsafeExecutor):
 
     RUN_SCRIPT_TEMPLATE: Template = JINJA_ENV.get_template("run_sandbox.sh.jinja")
 
-    async def _execute(self, _: str, __: Program, cwd: Path, ___: ComputeContext) -> ExecutorResult:
-        if not Path(CONTY_PATH).exists():
-            raise RuntimeError(f"Conty binary not found at {CONTY_PATH}!")
+    def _cmd(self, cwd: Path) -> tuple[list[str], dict[str, str]]:
+        # if not Path(CONTY_PATH).exists():
+        #     raise RuntimeError(f"Conty binary not found at {CONTY_PATH}!")
 
         # NOTE: `uv` binary is assumed to be stored under `~/.cargo/bin/`
         # We are using `uv` as the environment manager and program runner
         uv_path = Path("~/.cargo/bin/uv").expanduser()
         # NOTE: We need to bind the uv cache folder to access uv-managed python executables
         uv_cache_path = Path("~/.local/share/uv").expanduser()
 
-        async with self.lock:
-            exec_proc = await asyncio.create_subprocess_shell(
-                shlex.join(
-                    [
-                        CONTY_PATH,
-                        "--bind",
-                        str(cwd.absolute()),
-                        str(cwd),
-                        "--ro-bind",
-                        *([str(uv_path)] * 2),
-                        "--ro-bind",
-                        *([str(uv_cache_path)] * 2),
-                        str(cwd / self.ENTRYPOINT),
-                    ]
-                ),
-                stdout=asyncio.subprocess.PIPE,
-                stderr=asyncio.subprocess.PIPE,
-                env={
-                    **os.environ,
-                    # Conty specific environment variables
-                    "SANDBOX": "1",
-                    "SANDBOX_LEVEL": "1",
-                    "QUIET_MODE": "1",
-                    # NOTE: We need to unset VIRTUAL_ENV to prevent uv from using it
-                    "VIRTUAL_ENV": "",
-                },
-            )
-
-            stdout, stderr = await exec_proc.communicate()
-
-        exit_code_file = cwd / "exit_code"
+        return [
+            CONTY_PATH,
+            "--bind",
+            str(cwd.absolute()),
+            str(cwd),
+            "--ro-bind",
+            *([str(uv_path)] * 2),
+            "--ro-bind",
+            *([str(uv_cache_path)] * 2),
+            str(cwd / self.ENTRYPOINT),
+        ], {
+            # Conty specific environment variables
+            "SANDBOX": "1",
+            "SANDBOX_LEVEL": "1",
+            "QUIET_MODE": "1",
+            # NOTE: We need to unset VIRTUAL_ENV to prevent uv from using the wrong base python interpreter
```

Comment:
We might not need this anymore with newer versions of `uv`, will verify before removing. Previously, `VIRTUAL_ENV` was unset to prevent thrashing of `stderr` with its warnings. However, I no longer observe that anymore but it just be I suppressed `uv` logs via `--quiet`

_This applies to `UnsafeExecutor` as well_

Reply:
Will keep it for now, we still need this